.eslintrc.js

```diff
@@ -37,6 +37,13 @@ const defaultRules = {
   // Allow using any for rest arguments (i.e. `...any[]`)
   '@typescript-eslint/no-explicit-any': ['warn', { ignoreRestArgs: true }],
 
+  '@nrwl/nx/enforce-module-boundaries': [
+    2,
+    {
+      allowCircularSelfDependency: true,
+    },
+  ],
```

Comment:
Is this a temporary fix while other refactoring is being done? Or has it more to do with a "false positive" due to the `@/` -> `@api/` change? I just noticed Nx does recommend using relative paths within an app/library and using `@project/lib` style imports for library imports.

Reply:
The linting rule source code documents the `allowCircularSelfDependency` as follows: `Projects should use relative imports to import from other files within the same project. Use "./path/to/file" instead of import from "{{imp}}"`

I had to relax this rule because I was getting linting rule failures for existing code that I hadn't written complaining about using module aliases to reference the app that you are currently in.  For example, if you had a module in the api app that had an import that used `@api/[...]`, the linter would complain and tell you to replace the alias with a relative path.

I don't have an opinion about relative paths versus aliases.  Actually when I made these changes before on "the client's" fork of this repo, I had originally just fixed all the linting errors.  But then in that PR people said they liked having the ability to use `@api/` within the api project so I had to revert the changes and relax the linting rule.  That took me quite a while to do so this time I just left it be.